docs/sources/panels/thresholds.md

```diff
@@ -0,0 +1,51 @@
+title = "Thresholds"
+type = "docs"
+[menu.docs]
+identifier = "thresholds"
+parent = "panels"
+weight = 300
++++
+
+# Thresholds
+
+Thresholds set the color of either the value text or the background depending on conditions that you define.
+
+You can define thresholds one of two ways:
+* **Absolute** thresholds are defined based on a number. For example, 80 on a scale of 1 to 150.
+* **Percentage** thresholds are defined relative to minimum and maximums. For example, 80 percent.
+
+You can apply thresholds to the following visualizations:
+* Stat
+* Gauge
+* Bar gauge
+* Table
+
+## Default thresholds
+
+On visualizations that support it, Grafana sets default threshold values of:
+* 80 = red
+* Base = green
+* Mode = Absolute
+
+The **Base** value represents minus infinity. It is generally the “good” color.
+
+## Add a threshold
+
+You can add as many thresholds to a panel as you want. Grafana automatically sorts thresholds from highest value to lowest.
+
+1. Navigate to the panel you want to add a threshold to.
```

Comment:
Shit, this flow is not valid for Graph panel, as it uses legacy thresholds editor:/ we need to document that too

Reply:
Why can't we update it to the new editor?

Reply:
Can we just update the Graph panel to the new editor?

Reply:
nope, it's an angular panel, so we have to wait